src/app/components/styles/app.scss

```diff
@@ -15,3 +17,24 @@
 .loader-container + .loader-container {
   display: none;
 }
+
+.card__header {
+  padding-top: $global-padding;
```

Comment:
Should this be defined in Franklin instead?

Reply:
Actually I had forgotten to make it more specific, it is now covering headers but only containing small text